brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Block Categorization - Updated admin category sequence as per QA suggestion.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/1e3f868b-d473-4475-beaa-5c7da1c592a3?redirect=true&view=6d3bcc3d-b047-4739-8e05-d9621e9f9445&t=attributes

### Screenshots

- https://share.getcloudapp.com/wbuz6zBE

### Types of changes




- Improvement

### How has this been tested?

- Go to the admin dashboard.
- Check All, Creative, Content, SEO, Social, Post, Form, Extention Tabs, and Block categories as per tabs.

### Checklist:
